charts/k8s-aws-ebs-tagger/templates/deployment.yaml

```diff
@@ -83,6 +83,16 @@ spec:
               port: http
           resources:
             {{- toYaml .Values.resources | nindent 12 }}
+        {{- if .Values.volumeMounts }}
+        volumeMounts:
```

Comment:
The `volumeMounts` should be part of the container not the outer spec (2 spaces more). It should be at the same indentation level as ports, resources, etc, right?

Reply:
I've fixed the indentation and combined the commits into one.